chart/templates/services/api/ingress.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if and .Values.global.huggingface.ingress.enabled .Values.ingress.enabled .Values.api.ingress.enabled -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  {{- $annotations := fromYaml (include "datasetsServer.instance.ingress.annotations" (dict "instance" .Values.api "context" $ )) }}
+  annotations: {{ toYaml $annotations | nindent 4}}
+  labels: {{ include "labels.api" . | nindent 4 }}
+  name: "{{ include "name" . }}-api"
+  namespace: {{ .Release.Namespace }}
+spec:
+  rules:
+    - host: {{ include "datasetsServer.ingress.hostname" . }}
+      http:
+        paths:
+          - backend:
+              service:
+                name: "{{ include "name" . }}-api"
+                port:
+                  name: http
+            path: /
+            pathType: Prefix
+{{- include "ingress.tls" (merge (dict "annotations" $annotations) $ ) | indent 2}}
+{{- end }}
```

Comment:
we have to forbid /metrics here, I think

Reply:
The best way to do that is too expose metrics on another port and not expose it.
But let's try to block the path at the ALB level.

Reply:
hmmm ok. https://github.com/huggingface/dataset-viewer/issues/2812